### PR TITLE
Query capability - Added Query Method for product name and supplier name filtering 

### PR DESCRIPTION
**Pull request by marina2232**

### CODE REVIEW

**Comment by jrose023** [comment]
@stefanisandridge any idea why we arent seeing codecov show up in these checks?

**Comment by stefanisandridge** [comment]
Let me take a look. There should be 4 successful checks (2 Travis CI and 2 Codecov). Today it now only shows 1 check for Travis CI. Not sure what/how anything changed. Tagging @aganeesh as well so he sees the comment.

**Comment by stefanisandridge** [comment]
Ignore my previous comment. We were missing code coverage details from the requirements.txt file. It was already included during class demo which is why it wasn't linking. Aneesh and I created a new issue #41 to add the 2 lines of code and now the tests should work. Thank you!

File: service/models.py [hunk 1]
```diff
@@ -125,11 +125,21 @@ def find_or_404(cls, inventory_id):
...
 
         Args:
             product_name (string): the name of the InventoryItems you want to match
         """
         logger.info("Processing name query for %s ...", product_name)
         return cls.query.filter(cls.product_name == product_name)
+
+    @classmethod
+    def find_by_supplier_name(cls, supplier_name):
+        """Returns all InventoryItems with the given name
```
Comment (jrose023) [inline service/models.py]: should say supplier name not name

Comment (jrose023) [reply service/models.py]: @neeharikatummala

File: service/routes.py [hunk 2]
```diff
@@ -77,7 +77,17 @@ def list_inventory_items():
...
     app.logger.info("Request for inventory list of all products")
 
-    all_inventory_items = InventoryItem.all()
+    all_inventory_items = []
+
+    supplier_name = request.args.get("supplier_name")
+    product_name = request.args.get("product_name")
+    if supplier_name:
+        all_inventory_items = InventoryItem.find_by_supplier_name(supplier_name)
+    elif product_name:
```
Comment (jrose023) [inline service/routes.py:87]: add an elif for supplier_id since now we have that new functionality

**Review by aganeesh — APPROVED** [review]
lgtm

**Pull request merged**